README.md

```diff
@@ -9,34 +9,20 @@ You are supposed to build a website which shows random "fortune" messages, just
 like the one you find in fortune cookies. Note that our fortune messages also
 include a title.
 
-Each fortune message may receive anonymous comments, the comment must be at
-least 10 character long and at most 255 characters long.
+Each fortune message may receive comments, the comment must be at least 10
+character long and at most 255 characters long.
 
-We don't believe you will be needing any layouts (.psd) to complete this test.
-
-You should follow the following screen design:
-
-![](resources/screen.png)
+Just users that have an account can create a "fortune" messages and can add some
```

Comment:
Acho que é melhor usar:

`Only authenticated users should be able to post or comment on "fortune" messages.`